### PR TITLE
[MTM-58756] Release notes: removal deprecated JWT with Cumulocity pro…

### PR DESCRIPTION
**Pull request by mbak-c8y**
On Jun 3, an Announcement was published regarding **plans**  removal of JWT with Cumulocity provider: https://github.com/Cumulocity-IoT/c8y-docs/pull/1915

This is an announcement to inform that it has been done.

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58756
+version: 10.20.596.0
+---
+Deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md]:
```suggestion
The deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.
```

File: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This change will only impact you if your organization used an early version of SSO on {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md]:
```suggestion
This change will only impact you if your organization used an early version of single sign on (SSO) on {{< product-c8y-iot >}}.
```

File: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+More details can be seen in the early published [announcement](https://cumulocity.com/docs/change-logs/?#cumulocity-10-20-removal-of-deprecated-jwt-login-mode)
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md]:
```suggestion
For details, refer to the earlier [announcement](https://cumulocity.com/docs/change-logs/?#cumulocity-10-20-removal-of-deprecated-jwt-login-mode).
```

File: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+date: 2024-10-15
+title: Removal of deprecated JWT login mode
+product_area: Platform services
+change_type:
+  - value: change-inv-3bw8e
+    label: Announcement
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md:7]: This is not an announcement, but an improvement I would say.

Comment (mbak-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md:7]: Within a team, we talked that, it should go as an announcement. We announce that deprecated JWT can not be used anymore.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58756
+version: 10.20.596.0
+---
+The deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.
```
Comment (kowa-c8y) [inline content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md:17]: Minor: The deprecated JWT authentication method has been removed from {{< product-c8y-iot >}}. - that seems to be clearer for me but I see that @BeateRixen approved that so I put it as optional :)

**Review by kowa-c8y — APPROVED** [review]

**Pull request merged**